contracts/lpp/src/contract/lender.rs

```diff
@@ -125,6 +144,7 @@ mod test {
     const BASE_INTEREST_RATE: Percent = Percent::from_permille(70);
     const UTILIZATION_OPTIMAL: Percent = Percent::from_permille(700);
     const ADDON_OPTIMAL_INTEREST_RATE: Percent = Percent::from_permille(20);
+    const DEFAULT_MIN_UTILIZATION: BoundToHundredPercent = BoundToHundredPercent::ZERO;
 
     #[test]
     fn test_deposit() {
```

Comment:
[nice to have, optional for this PR] please provide short and single-purpose tests

Reply:
I'll split this one too.